src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,10 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { UserInfoProps } from '../../Types';
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
```

Comment:
Don't forget to extract properties here, it's just a nice little approach to get used to:
`{ user: { email, name } }`